applications_tests/lethe-fluid-matrix-free/CMakeLists.txt

```diff
@@ -2,11 +2,13 @@ set(TEST_TARGET lethe-fluid-matrix-free)
 
 string(TOLOWER ${CMAKE_BUILD_TYPE} _build_type)
 
+file(COPY cylinder_structured.msh DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/cylinder_kelly_estimator.${_build_type}")
+
```

Comment:
@lpsaavedra 
Generally to copy the files in the example we use:
file(COPY taylorCouette.msh DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
and this would copy the file in which the case is running.
I am unsure in the present case where the file is being copied when you run it like this?